ydb/docs/en/core/concepts/backup-collections.md

```diff
@@ -0,0 +1,308 @@
+# Backup collections
```

Comment:
From neighboring articles, it sounds like backup collections are a schema object kind, thus this article belongs to the https://ydb.tech/docs/en/concepts/datamodel/?version=v25.2 folder.